### ISSUE TITLE
Adding users/passwords to the Postgres

### ISSUE THREAD

**Issue by ravinder-Olivier** [ISB3DPRINTERS/teachers-console#5]
The method used in the example script provided doesn't work ._.

**Comment by ravinder-Olivier** [comment]
@RedstoneWizard08

**Comment by RedstoneWizard08** [comment]
I'll take a look soon, sorry I've been busy

**Comment by ravinder-Olivier** [comment]
Yeah, no rush, just wanted to have it documented. I'm working on making a full project roadmap

**Comment by RedstoneWizard08** [comment]
I think I found the issue:
Before starting the server, run these commands:

```sh
cd server
cargo install diesel_cli --no-default-features --features postgres
diesel setup
diesel migration run
```

Then it should work. I think the issue was that the database wasn't set up correctly, I missed putting that step in the instructions.

**Comment by ravinder-Olivier** [comment]
I tried all of those, uninstalling and reinstalling diesel_cli, following the instructions, and I'm still left with the attached screenshot
![Screen Shot 2023-11-03 at 9 26 18 PM](https://github.com/ISB3DPRINTERS/teachers-console/assets/77025041/7b85da9b-a0a5-4b2e-8ca7-816c896f2083)

**Comment by RedstoneWizard08** [comment]
You're not using the username and password you registered with. That's why. It's working as intended.

**Comment by ravinder-Olivier** [comment]
I'm not so sure, it's giving me errors when I try to register or login, and when I try on the webclient I'm stuck on the login page.
![Screen Shot 2023-11-04 at 10 17 31 PM](https://github.com/ISB3DPRINTERS/teachers-console/assets/77025041/cdbb0eee-841f-4e09-8cb7-a1cbc67798d9)

**Comment by RedstoneWizard08** [comment]
Calling register again will return invalid because the user already exists.

**Comment by RedstoneWizard08** [comment]
And you're stuck because it doesn't do anything after login, you need to implement that

**Comment by ravinder-Olivier** [comment]
1.
 ![Screen Shot 2023-11-05 at 12 35 21 PM](https://github.com/ISB3DPRINTERS/teachers-console/assets/77025041/c0aa45a9-da42-42ef-9001-e6f7bac3775e)
It happens when I try with a new user too, and I supposed to do a different type of request? 
2. Oh, ok, I thought it took to the Account page

**Comment by RedstoneWizard08** [comment]
ok idk why thats happening now

**Comment by RedstoneWizard08** [comment]
if you cant figure this out just write it in js idk its ur project

**Comment by RedstoneWizard08** [comment]
sorry
